lib/template.html

```diff
@@ -51,7 +51,11 @@
 
 {% if site.twitter %}
   {% if seo_tag.image %}
-    <meta name="twitter:card" content="summary_large_image" />
+      {% if site.twitter.smallSummary %}
```

Comment:
Alternatively, perhaps something like `<meta name="twitter:card" content="{{ site.twitter.card | default: "summary_large_image" }}" />` may be more intuitive to end users?